pandas/_libs/tslibs/dtypes.pyx

```diff
@@ -504,6 +514,45 @@ cdef NPY_DATETIMEUNIT get_supported_reso(NPY_DATETIMEUNIT reso):
     return reso
 
 
+cdef npy_datetimestruct dts_us_min, dts_us_max
+pandas_datetime_to_datetimestruct(
+    np.iinfo(np.int64).min + 1, NPY_DATETIMEUNIT.NPY_FR_us, &dts_us_min
+)
+pandas_datetime_to_datetimestruct(
+    np.iinfo(np.int64).max, NPY_DATETIMEUNIT.NPY_FR_us, &dts_us_max
```

Comment:
util.INT64_MAX

Reply:
if you implement this in np_datetime.pyx you can reuse _NS_MIN_DTS, _NS_MAX_DTS, etc